pom.xml

```diff
@@ -3,9 +3,9 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>life.qbic</groupId>
-	<artifactId>templateQbicPortlet</artifactId>
+	<artifactId>qbic_portlet_template</artifactId>
 	<version>1.0</version>
-	<name>templateQbicPortlet</name>
+	<name>QBiC Portlet Template</name>
```

Comment:
I changed the name to the same name as in the liferay files for a better consistency.